packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -88,6 +88,11 @@ export class AtomicSearchInterface {
    */
   @Prop() public reflectStateInUrl = true;
 
+  /**
+   * The CSS selector for the container where the interface will scroll back to.
```

Comment:
Does Stencil resolve the default value from the code? Or does the default need to be in the annotation?

Reply:
The doc resolves the default